content/en/docs/topics/permissions_sql_storage_backend.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Permissions management for SQL storage backend"
+description: "Get to know how to properly setup permissions when using SQL storage backend."
+aliases: ["/docs/permissions_sql_storage_backend/"]
+---
+
+This document aims to provide guidance to Helm 3 users for setting up and managing permissions properly when using the SQL storage backend.
+
+## Introduction
+
+To handle permissions, Helm leverages the RBAC feature of Kubernetes. When using the SQL storage backend, Kubernetes' roles can't be used to determine whether or not an user can access a given resource. We'll here discuss how to properly create and manage these permissions.
+
+## Initialization
+
+The first time the Helm CLI will make connect to your database, the client will make sure that it was previously initialized. If it was not, it will take care of running the migration scripts. This initialization requires admin privileges on the public schema, or at least to be able to:
+
+* create a table
+* grant privileges on the public schema
+
+After the migration was run against your database, all the other roles can use the client.
+
+## Grant privileges to a non admin user
+
+To manage permissions, the SQL backend driver leverages the [RLS](https://www.postgresql.org/docs/9.5/ddl-rowsecurity.html)(Row Security Level) feature of PostgreSQL. RLS allows all users to be able to read/write from/to the same table, without being able to manipulate the same rows if they were not explicitely allowed to. By default, any role that has not been explicitely granted with the right privileges will always get an empty list when running `helm list` and will not be able to get nor to modify any resource in the cluster.
```

Comment:
if they were not explicitely --> if they are not explicitly

always get --> always return

not be able to get nor to modify --> not be able to retrieve or modify